docs/release-notes/release-notes-1.0.0.md

```diff
@@ -0,0 +1,104 @@
+JJDlQJhBtioUaJ2jBBXEITzhNQ9DA/x+iUN7rQFNhKsMeczEErDmKucGrPSqQKiG
+Hz7c20lVnrT5EGdEL1bBPzSY7SJKkzn0rgsAxPHFxZlSkY/AFhn+/py/0+Tk1Rjq
+7ullWCiAtBpOtIySGv8wnJHNSXVRE3kU5vPOXeF0YVqhjgIyVK+1jiXErSRQaSn1
+pkpeNavFQQ0CrJlEeKFWicvBkbSfuhDArDeyq8PMdGCaD2pvMEla5VvTi1A3PLkB
+jQRky2shAQwAt01gHBk+g0UP5WUcJgL+FaC0k4Lpk7mvd3ZKuVLObyRyEgNnsTLS
+sgEUNBbaQ+y5XTV7O6VsLWQRNUQiSvy3bOPh+jYellTaJA1XVdM/07dGrVuWmAY+
+wAi8i/NIJurDXhasU8e6Hj/RnQzYVXweYgbuH31c/9RTaU3ndjEBBUE4jqImsYbs
+zG5HxLo154CUzH88YsCf5ZN6Jzl3GLo+213Y0dGT7EHzDpxDtLPzJuo8lnOEqR4/
+5RistNF1fVGl4zma5pUk8ZerRT/mOPm+xAbWXeR6nJEXNI69g3ixPLOdM44jqHbo
+XIpB4FQWJJAtrOLJqM65CPDhWIGs6fvG2U7tv66XVvrUKTBOCfOceGyJEGyKWGX6
+kDECEWFTOdnalsGhX3PUriejhbCNQbkoK6a9PcRIM/VaK++I9eaytz1lShB03Je2
+uXBtzf3uTlc2e/2WZvMmBictoXEZ22WkJh8dDRaKcuxfjkft3DovbcNgP1Yl332b
+oumpQa/FcsdpABEBAAGJAbwEGAEKACYWIQRcJr/sbCRmpSjVVRzQWsdPaJduUgUC
+ZMtrIQIbDAUJA8JnAAAKCRDQWsdPaJduUi+dDAC3mm2EvHXUYxGPvYb6GqfqseDQ
+4rba+jjD6TUaAJVitpTY3CoKKBrdEWRZPR5+WSS/Hx75uAVS84g+3AZza5O4f100
+spZkQtmOxQ4lHVtSZgaj/DR1dgvunHF5bqbiIq1VNOvRGcXafxhKvCksM/aKST6f
+lxYeC8RQAaK8QUJCLqGBR6aiCkFa4/+t95rUEEFrRuukYXm77cfhFlWfiDk5QB5a
+gBjYVCa6INCaEw57dqB3CxBQloIZ4m7PY0ofYiKVD+PWidwztbfO4ee4MeaXVo6z
+NcUVH+fzcCZGgDO1sTuDOzOAdx5uvPmpyzMXzDAe5VYgLoHrsjDhJWsB9YZZjKhx
+/PN+YS3KGsrD8HMgNhAdn3/QkLD2OKl3/sW5kmuFmlywdLkocl8FmZlGyRd/XxVD
+u374UjAHT4NkQzbNV9DPKFeWk8yQq/xVj7wSezndFb/+YuGyz5Skji7mKTuoCjWS
+pawty5DgDQwzHHdhqGlT7Dnb/R9QMEyPvK12HaQ=
+=dt9F
+-----END PGP PUBLIC KEY BLOCK-----
+```
+
+Simply copy that entire text, save it as `key.txt`, and open a terminal.
+
+In the terminal, use the cd command to navigate to the directory path where you’ve saved `key.txt`. Then use the `gpg --import` command.
+
+(Note that Windows uses \ for directory paths, but Linux/macOS use /)
+
+```
+$ cd /path/to/the/key
+$ gpg --import < key.txt
+```
+
+Example output:
+
+```
+gpg: key D05AC74F68976E52: "Cryptopower <release@cryptopower.dev>" imported
+gpg: Total number processed: 1
+gpg:              unchanged: 1
+```
+
+You can then delete key.txt.
+
+Once you have the required PGP keys, you can verify the release with:
+
+`gpg --verify cryptopower-v1.0.0-manifest.txt.asc`
+
+You should see the following if the verification was successful:
+
+```
+gpg: assuming signed data in 'cryptopower-v1.0.0-manifest.txt'
+gpg: Signature made Thu 03 Aug 2023 17:08:06 WAT
+gpg:                using RSA key 5C26BFEC6C2466A528D5551CD05AC74F68976E52
+gpg:                issuer "release@cryptopower.dev"
+gpg: Good signature from "Cryptopower <release@cryptopower.dev>" [ultimate]
+```
+
+That will verify the signature of the manifest file, which ensures integrity and authenticity of the archive you've downloaded locally containing the binaries. Next, depending on your operating system, you should then re-compute the `sha256` hash of the archive with `sha256sum <filename>`, compare it with the corresponding one in the manifest file, and ensure they match exactly.
```

Comment:
> and ensure they match exactly

Can drop "exactly"? I think.

Reply:
i think "exactly" is an emphasis while still being correct English

Reply:
np, we can keep it for "emphasis".